schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+-- schemas
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INT REFERENCES patients(id),
+    status VARCHAR(500)
+);
+
+CREATE TABLE invoices (
+    id SERIAL PRIMARY KEY,
+    total_amount VARCHAR(100),
+    generated_at TIMESTAMP,
+    paid_at TIMESTAMP,
```

Comment:
- [ ] To ensure consistency in the requirement and the implementation so as to prevent query issues, kindly consider ignoring the grammatical correctness of `paid_at` and use `payed_at` as you have in the ERD defined by the user. Except where you have confirmed with your client, it is important to implement the user's story as requested.